packages/maas/bff/internal/models/namespace.go

```diff
@@ -6,9 +6,9 @@ type NamespaceModel struct {
 }
 
 func NewNamespaceModelFromNamespace(name string) NamespaceModel {
-	displayName := name // For now, use name as display name, but this can be customized later
+	displayName := name + "-maas" // For now, use name as display name, but this can be customized later
```

Comment:
This was just to clearly differentiate the data is coming from MaaS BFF.
We, anyway may want to remove this code, assuming existing TS back-end may already have this covered (and I don't think we need this for MaaS).